weaver/processes/wps3_process.py

```diff
@@ -45,15 +45,17 @@
 
 LOGGER = logging.getLogger(__name__)
 
-REMOTE_JOB_PROGRESS_PROVIDER = 1
-REMOTE_JOB_PROGRESS_PREPARE = 2
-REMOTE_JOB_PROGRESS_DEPLOY = 3
-REMOTE_JOB_PROGRESS_VISIBLE = 4
-REMOTE_JOB_PROGRESS_READY = 5
-REMOTE_JOB_PROGRESS_EXECUTION = 9
-REMOTE_JOB_PROGRESS_MONITORING = 10
-REMOTE_JOB_PROGRESS_FETCH_OUT = 90
-REMOTE_JOB_PROGRESS_COMPLETED = 100
+
+class Wps3RemoteJobProgress(WpsRemoteJobProgress):
+    PROVIDER = 1
+    # PREPARE = 2
+    DEPLOY = 3
+    VISIBLE = 4
+    # READY = 5
+    EXECUTION = 9
+    MONITORING = 10
+    FETCH_OUT = 90
+    # COMPLETED = 100
```

Comment:
It inherits from the base one. 
Would it be clearer if I override it anyway although same value?
I see a marker in my IDE, but hard to see with plain code.

An yeah! The tests suite was life saver for this. I wouldn't even have tried without it.

Reply:
Hum ok, so the ones seen here have actually a different value... I think that yes it would be clearer and safer... What if someone change the value in the base one without checking if it still make sense in the various implementation?

Reply:
They were the same values (literal copy-paste), but I agree it is safer to override explicitly just in case of modification in the base definition later on.